src/money.js

```diff
@@ -11,21 +13,8 @@ export default class Money {
   }
 
   static buildFromString(stringNumber) {
-    // remove everything (comma, $, etc.) except the number and decimal in between
-    // - decimal optional, no decimal = dollars
-    const number = stringNumber.replace(/,/g, "").match(/[0-9]*\.?[0-9]*$/, "");
-    if (number == null) return 0; // if num is other string
-    const numbers = number[0].split(".");
-    const dollars = parseInt(numbers[0], 10) || 0;
-    var centString = numbers[1]?.substring(0, 2);
-    // padding centString if it's only a single digit
-    // so that .2 is counted as 20 cents instead of 2
-    // Also truncates anything past 2 decimal places (no rounding, only truncating)
-    if (centString?.length == 1) {
-      centString = `${centString}0`;
-    }
-    const cents = centString != null ? parseInt(centString, 10) : 0;
-    return dollars * 100 + cents;
+    const currencyValue = currency(stringNumber)
+    return currencyValue.dollars() * 100 + currencyValue.cents();
```

Comment:
currencyValue.value seems to be the same thing?

Reply:
would just write `return currency(stringNumber).value` if that works.

Reply:
We're after the value in cents here, so it'll have to be `return currency(stringNumber).value * 100;` but that should be fine